### ISSUE TITLE
Quick start failed

### ISSUE THREAD

**Issue by mulka** [coiled/coiled-issues#72]
I tried the quick start, but got an exception on the login step:

Exception: {'detail': 'Authentication credentials were not provided.'}

**Comment by mrocklin** [comment]
Thanks Kyle.  This was due to a botched release.  This will be handled
shortly.

**Comment by jrbourbeau** [comment]
Yeah, thanks for raising an issue. We just released `coiled` 0.0.25 to PyPI (should be on conda-forge in an hour or so) which should fix that. Could you upgrade to the latest `coiled` release and try the quick start again?

**Comment by mulka** [comment]
Ok, I have a cluster now, but the dashboard_link it gave me just shows the coiled sidebar. Nothing in the main content area.

**Comment by jrbourbeau** [comment]
What is the `dashboard_link` you're seeing? I get something along the lines of https://cloud.coiled.io/dashboard/920/status

**Comment by jrbourbeau** [comment]
I should also mention that the above link points to a Dask dashboard for me (screenshot below)

<img width="1552" alt="Screen Shot 2020-09-22 at 1 21 11 PM" src="https://user-images.githubusercontent.com/11656932/93921699-83a43880-fcd6-11ea-95b5-009d4b35f5b2.png">

**Comment by necaris** [comment]
@mulka I too am unable to replicate :disappointed: Do you happen to have a screenshot of what it's showing you, as well?

For reference, this is what I see:
![2020-09-22_14-11](https://user-images.githubusercontent.com/460769/93922595-23b28f80-fce0-11ea-83c7-bd90ac57baf5.png)

Clicking on the eye icon next to the status shows me:
![2020-09-22_14-11_1](https://user-images.githubusercontent.com/460769/93922632-30cf7e80-fce0-11ea-8376-f7de39cd8d75.png)

The link given to me in the Python code also seems to work for me:
![2020-09-22_14-11_2](https://user-images.githubusercontent.com/460769/93922663-3fb63100-fce0-11ea-9be1-b62fcbc797cb.png)

**Comment by necaris** [comment]
There's one thing I can think of that I'm looking into now

**Comment by mulka** [comment]
I started a new cluster and it worked that time. Not sure why it didn't work before. Then I started a 3rd cluster and ran into this one time. After trying a couple times it started working again. Maybe it was just a timing thing? Like visiting the link or clicking the button right around when it is starting.
![Screen Shot 2020-09-22 at 2 52 19 PM](https://user-images.githubusercontent.com/51038/93924989-79d50200-fce3-11ea-9882-764cfdf0bb09.png)

**Comment by necaris** [comment]
@mulka the next time you run into this, would you be able to open your browser console and let us know if there are any errors in it? If you're able to reproduce it consistently one of us would love to get on a video chat and see what's going on, too.

**Comment by dantheman39** [comment]
@mulka It's been a while since there was any movement on this, I'm hoping it's not happening anymore (there have been a number of fixes and updates since this issue was raised), so am going to close the issue, but feel free to reopen if it happens again. Thanks for reporting!
